confidential_backend/config.py

```diff
@@ -5,6 +5,7 @@
 import os
 import redis
 
+AUTH_TOKEN_LOG_FILTER = os.getenv("AUTH_TOKEN_LOG_FILTER")
```

Comment:
Minor nitpick: if we ever want to override by env var, we'll want to pass them as a comma separated list
```suggestion
AUTH_TOKEN_LOG_FILTER = os.getenv("AUTH_TOKEN_LOG_FILTER").split(",") if "AUTH_TOKEN_LOG_FILTER" in os.env else None
```